auth-web/src/components/auth/common/DatePicker.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+    <v-card class="date-selection registration-date" elevation="6" ref="datePicker">
+        <v-row no-gutters>
+            <v-col class="picker-title" :class="{ 'picker-err': startDate === null && datePickerErr }" cols="6">
+                Select Start Date:
+            </v-col>
+            <v-col class="picker-title pl-4" :class="{ 'picker-err': endDate === null && datePickerErr }" cols="6">
+                Select End Date:
+            </v-col>
+        </v-row>
+        <v-row>
+            <v-col cols="6">
+                <v-date-picker color="primary" :max="endDate ? endDate : today" v-model="startDate" />
+            </v-col>
+            <v-col cols="6">
+                <v-date-picker color="primary" :min="startDate ? startDate : null" :max="today" v-model="endDate" />
+            </v-col>
+        </v-row>
+        <v-row no-gutters justify="end">
+            <v-col cols="auto pr-4">
+                <v-btn class="date-selection-btn bold" ripple small text @click="submitDateRange()">
+                    OK
+                </v-btn>
+                <v-btn class="date-selection-btn ml-4" ripple small text @click="resetDateRange()">
+                    Cancel
+                </v-btn>
+            </v-col>
+        </v-row>
+    </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
+import Vue from 'vue'
+
+@Component({})
+export default class DatePicker extends Vue {
+    @Prop({ default: '' }) setEndDate: string
+    @Prop({ default: '' }) setStartDate: string
+
+    private datePickerErr: boolean = false
+    private endDate = null
+    private startDate = null
+
+    private get defaultMonth () {
+      const todayDate = new Date()
+      return todayDate.toISOString().substring(0, 8)
```

Comment:
No moment here, not sure how this would react in different TZ.. but should be ok considering it's staff?

Reply:
I think so